@xen-orchestra/web/src/components/hosts/hostsTable.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <div class="host-table">
```

Comment:
```suggestion
  <div class="hosts-table">
```
the class name is not supposed to be the same as the component name?